CybeReason/malop-activity/_meta/smart-descriptions.json

```diff
@@ -25,6 +25,16 @@
     ],
     "relationships": []
   },
+  {
+    "value": "Update of a report on a malicious activity (see {event.url}): {event.reason}\naffected hosts: {cybereason.malop.counters.affected_machines}\naffected users: {cybereason.malop.counters.affected_users}",
+    "conditions": [
+      {
+        "field": "event.action",
+        "value": "Malop Modified"
+      }
```

Comment:
Can you add the fields using in the placeholder as conditions of the smart-description?
Thanks

Reply:
Updated. Thanks!